File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int n = arr1.length;
        int m = arr1[0].length;
        int[][] ret = new int[n][m];
        for(int i = 0; i < n; i++){
            for(int j = 0; j < m; j++){
                ret[i][j] = arr1[i][j] + arr2[i][j];
            }
        }
        return ret;
    }
}

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.py
def solution(arr1, arr2):
    row, col = len(arr1), len(arr1[0])
    ret = [[0] * col for _ in range(row)]
    for i in range(row):
        for j in range(col):
            ret[i][j] = arr1[i][j] + arr2[i][j]
    return ret
            